protocol/protobuf/group_chat_invitation.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package protobuf;
+
+message GroupChatInvitation {
+  // chat_id the ID of the private group chat the message belongs to, for query efficiency the chat_id is stored in the db even though the
+  // target message also stores the chat_id
+  string chat_id = 1;
+
+  string introduction_message = 2;
+
+  bool rejection = 3;
```

Comment:
What is the default state of a group chat invitation?

If the state is false, how does the application distinguish the default state from a rejected state?
If the state is true, how does the application distinguish the default state from an accepted state?

Reply:
there is no accepted state, only two states requested and rejected